mirdata/datasets/dagstuhl_choirset.py

```diff
@@ -0,0 +1,411 @@
+    capture the individual singers’ voices.
+
+    For more details, we refer to:
+    Sebastian Rosenzweig (1), Helena Cuesta (2), Christof Weiß (1),
+    Frank Scherbaum (3), Emilia Gómez (2,4), and Meinard Müller (1):
+    Dagstuhl ChoirSet: A Multitrack Dataset for MIR Research on Choral Singing.
+    Transactions of the International Society for Music Information Retrieval,
+    3(1), pp. 98–110, 2020.
+    DOI: https://doi.org/10.5334/tismir.48
+
+    (1) International Audio Laboratories Erlangen, DE
+    (2) Music Technology Group, Universitat Pompeu Fabra, Barcelona, ES
+    (3) University of Potsdam, DE
+    (4) Joint Research Centre, European Commission, Seville, ES
+"""
+import csv
+
+import librosa
+import numpy as np
+
+from mirdata import download_utils, jams_utils, core, annotations, io
+
+BIBTEX = """
+@article{RosenzweigCWSGM20_DCS_TISMIR,
+author    = {Sebastian Rosenzweig and Helena Cuesta and Christof Wei{\ss} and Frank Scherbaum and Emilia G{\'o}mez and Meinard M{\"u}ller},
+title     = {{D}agstuhl {ChoirSet}: {A} Multitrack Dataset for {MIR} Research on Choral Singing},
+journal   = {Transactions of the International Society for Music Information Retrieval ({TISMIR})},
+volume    = {3},
+number    = {1},
+year      = {2020},
+pages     = {98--110},
+publisher = {Ubiquity Press},
+doi       = {10.5334/tismir.48},
+url       = {http://doi.org/10.5334/tismir.48},
+url-demo  = {https://www.audiolabs-erlangen.de/resources/MIR/2020-DagstuhlChoirSet}
+}
+"""
+
+REMOTES = {
+    "full_dataset": download_utils.RemoteFileMetadata(
+        filename="DagstuhlChoirSet_V1.2.3.zip",
+        url="https://zenodo.org/record/4618287/files/DagstuhlChoirSet_V1.2.3.zip?download=1",
+        checksum="82b95faa634d0c9fc05c81e0868f0217",
+        unpack_directories=["DagstuhlChoirSet_V1.2.3"],
+    ),
+}
+
+DOWNLOAD_INFO = """
+Downloading dataset from Zenodo (5.1 GB)...
+"""
+
+LICENSE_INFO = """
+Creative Commons Attribution 4.0 International
+"""
+
+
+class Track(core.Track):
+    """Dagstuhl ChoirSet Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        audio_paths (list): paths to audio files
+        f0_paths (list): paths to F0-trajectories
+        score_paths (list): paths to time-aligned score representation
+        track_id (str): track id
+
+    Cached Properties:
+        f0 (annotations.F0Data): returns specified F0-trajectory
+        score (annotations.NoteData): returns time-aligned score representation
+    """
+
+    def __init__(self, track_id, data_home, dataset_name, index, metadata):
+
+        super().__init__(
+            track_id=track_id,
+            data_home=data_home,
+            dataset_name=dataset_name,
+            index=index,
+            metadata=metadata,
+        )
+
+        self.audio_paths = [
+            self.get_path(key)
+            for key in self._track_paths
+            if "audio" in key
+            if self.get_path(key)
```

Comment:
nit: for each of these, `if "audio" in key and self.get_path(key)`

Reply:
done!

Reply:
done!